### ISSUE TITLE
Deck Search Feature Request: Option to collapse decks with multiple versions

### ISSUE THREAD

**Issue by FullyWashable** [Kamalisk/arkhamdb#563]
Unless you are trying to track campaign progress, it is really annoying to have search results clogged with the myriad versions someone has made by saving every change they make to a public deck. It's easy to spot these all together when you sort by author reputation, ([Example: current results for Carolyn Fern](https://arkhamdb.com/decklists/find?faction=&cards%5B%5D=05001&author=&name=&sort=reputation&packs%5B%5D=1&packs%5B%5D=1-2&packs%5B%5D=76&packs%5B%5D=2&packs%5B%5D=4&packs%5B%5D=5&packs%5B%5D=8&packs%5B%5D=9&packs%5B%5D=10&packs%5B%5D=11&packs%5B%5D=12&packs%5B%5D=13&packs%5B%5D=14&packs%5B%5D=15&packs%5B%5D=16&packs%5B%5D=17&packs%5B%5D=18&packs%5B%5D=20&packs%5B%5D=21&packs%5B%5D=22&packs%5B%5D=23&packs%5B%5D=24&packs%5B%5D=25&packs%5B%5D=26&packs%5B%5D=29&packs%5B%5D=30&packs%5B%5D=31&packs%5B%5D=34&packs%5B%5D=35&packs%5B%5D=36&packs%5B%5D=37&packs%5B%5D=38&packs%5B%5D=40&packs%5B%5D=41&packs%5B%5D=42&packs%5B%5D=43&packs%5B%5D=44&packs%5B%5D=45&packs%5B%5D=52&packs%5B%5D=56&packs%5B%5D=57&packs%5B%5D=58&packs%5B%5D=59&packs%5B%5D=60&packs%5B%5D=61&packs%5B%5D=73&packs%5B%5D=74&packs%5B%5D=79&packs%5B%5D=80&packs%5B%5D=28&packs%5B%5D=33&packs%5B%5D=39&packs%5B%5D=54&packs%5B%5D=72&packs%5B%5D=47&packs%5B%5D=48&packs%5B%5D=49&packs%5B%5D=50&packs%5B%5D=51&packs%5B%5D=3&packs%5B%5D=6&packs%5B%5D=27&packs%5B%5D=32&packs%5B%5D=46&packs%5B%5D=55&packs%5B%5D=70&packs%5B%5D=19&packs%5B%5D=63&packs%5B%5D=64&packs%5B%5D=65&packs%5B%5D=66&packs%5B%5D=67&packs%5B%5D=68&packs%5B%5D=69&packs%5B%5D=7&packs%5B%5D=53&packs%5B%5D=62&packs%5B%5D=71&packs%5B%5D=75&packs%5B%5D=78).) With any other sort option, they are scattered throughout the results, and it makes it quite hard to know what you have already looked at, especially since many people might have decks with identical names. 

It would be really nice to have a checkbox that gives you the option to see such decks appear in the results only once, much like Craigslist lets you collapse identical posts of the same item for sale. Maybe there could be a tag for "multiple versions" or something like that on such collapsed entries, and the default would be to click through to the most recent version of the deck.
